scorecard.js

```diff
@@ -0,0 +1,39 @@
+class Scorecard {
```

Comment:
Good naming for all methods and variables. Not problem to understand what each method is doing. Of course you will have to include the handling of spares/strikes, which will make the code more complicated, but I like you tried to make it simple for the part that you did